src/transformers/models/layoutlm/modeling_tf_layoutlm.py

```diff
@@ -962,6 +964,7 @@ def call(
 
         >>> last_hidden_states = outputs.last_hidden_state
```

Comment:
Can you add an expected output shape here?

Reply:
same comment as above (prefer to do that in a separate change)